### PR TITLE
Address several post-commit comments. NFC

### PR DESCRIPTION
**Pull request by nkcsgexi**

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

**Review by compnerd — APPROVED** [review]
Thanks!

File: lib/IDE/Formatting.cpp [hunk 1]
```diff
@@ -575,7 +575,7 @@ class FormatWalker : public SourceEntityWalker {
       };
 
       if (auto AE = dyn_cast_or_null<ApplyExpr>(Node.dyn_cast<Expr *>())) {
-        // PrefixUnaryExpr shouldn't be syntacticly considered as a funtion call
+        // PrefixUnaryExpr shouldn't be syntacticly considered as a function call
```
Comment (dan-zheng) [inline lib/IDE/Formatting.cpp]:
```suggestion
        // PrefixUnaryExpr shouldn't be syntactically considered as a function call
```

**Review by dan-zheng — APPROVED** [review]

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

File: lib/IDE/Formatting.cpp (hunk 1 above)
Comment (nkcsgexi) [reply lib/IDE/Formatting.cpp]: Done

Comment (nkcsgexi) [reply lib/IDE/Formatting.cpp]: Done

**Pull request merged**